courses/free-courses-en.md

```diff
@@ -802,6 +803,10 @@
 
 > :information_source: See also &#8230; [C#](#csharp)
 
+### Networking
+
+* [Computer Networks | Tanenbaum](https://media.pearsoncmg.com/ph/streaming/esm/tanenbaum5e_videonotes/tanenbaum_videoNotes.html) - Pearson)
```

Comment:
Don't invent titles! Use the title given by the resource. 
Please add instructor names where appropriate.
Escape pipe characters

Reply:
I am sorry eric , i don't quite understand what you meant by inventing titles. There was no section for networking resources so i added one. And the title of the course is the same as on Pearson media website. 
Here is the attached screenshot. 





![Screenshot_2022-10-04-07-29-13-90.jpg](https://user-images.githubusercontent.com/76654888/193718370-33a93d4c-0f8f-41a3-ba29-9e22edb09cf3.jpg)